### PR TITLE
EWPP-1126: Drupal 9 compatibility fixes.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,20 +6,19 @@
...
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.7",
-        "php": ">=7.2"
+        "drupal/core": "^8.9 || ^9.1",
+        "php": ">=7.3"
     },
     "require-dev": {
-        "composer/installers": "~1.5",
+        "composer/installers": "^1.11",
```
Comment (sergepavle) [inline composer.json:13]: We can keep `~` for this constraint
```suggestion
        "composer/installers": "~1.11",
```

File: composer.json [hunk 2]
```diff
@@ -6,20 +6,19 @@
...
         "drupal/config_devel": "~1.2",
-        "drupal/core-composer-scaffold": "^8.8",
         "drupal/drupal-extension": "~4.0",
-        "drush/drush": "~9.0@stable",
-        "guzzlehttp/guzzle": "~6.3",
-        "openeuropa/code-review": "~1.0.0-beta2",
-        "openeuropa/drupal-core-require-dev": "^8.7",
-        "openeuropa/task-runner": "~1.0.0-beta5",
-        "phpunit/phpunit": "~6.0"
+        "drush/drush": "^10.3",
```
Comment (sergepavle) [inline composer.json:17]: We can keep `~` for this constraint as was before.
```suggestion
        "drush/drush": "^10.3",
```

File: drupal_module_template.info.yml [hunk 3]
```diff
@@ -2,5 +2,6 @@ name: OpenEuropa template
 description: OpenEuropa Drupal module template.
 package: OpenEuropa
 
+core_version_requirement: ^8 || ^9
```
Comment (sergepavle) [inline drupal_module_template.info.yml:5]: I suppose it should be like that:
```suggestion
core_version_requirement: ^8.9 || ^9.1
```
also looks like we have to remove deprecated for 8.9 already key `core` according to this document: https://www.drupal.org/node/3070687.

**Review by sergepavle — CHANGES_REQUESTED** [review]

**Review by sergepavle — CHANGES_REQUESTED** [review]
Also, the README file has to be updated regarding the introduced runner package for symlinks. As done in other components.

File: drupal_module_template.info.yml (hunk 3 above)
Comment (sergepavle) [reply drupal_module_template.info.yml]: Drone is failed because you need to remove the deprecated `core` key.

**Review by 22Alexandra — DISMISSED** [review]

**Review by sergepavle — APPROVED** [review]

**Review by upchuk — CHANGES_REQUESTED** [review]
Missing the quotes around "%paths.base%/tests/features" in the behat.yml file.

**Comment by upchuk** [comment]
> Missing the quotes around "%paths.base%/tests/features" in the behat.yml file.

Nevermind, I saw wrong

**Review by upchuk — APPROVED** [review]

**Review by ademarco — APPROVED** [review]

**Pull request merged**